File: C/level1/5 - Tests et conditions/1 - Transport des bagages.c
//coded by cybereagle2001
#include <stdio.h>

int main()
{
   int nbPaquets, poidsPaquet;
   scanf("%d %d", &nbPaquets, &poidsPaquet);
   
   if (nbPaquets * poidsPaquet > 105)
   {
      printf("Surcharge !\n");
   }
}

File: java/level 1/5- Tests et conditions/1- Transport des bagages.java
/** coded by cybereagle2001*/

import algorea.Scanner;

class Main{
   static Scanner input= new Scanner(System.in);
   public static void main(String[] args){
      int nbPaquet= input.nextInt();
      int poids= input.nextInt();
      int j=nbPaquet*poids;
      if (j> 105 ){
         System.out.println("Surcharge !");
      }
   } 
}
